features/test/src/test.ts

```diff
@@ -5,11 +5,19 @@ let quads: Array<RDF.Quad> = [];
 let pod: Pod;
 let polyIn: PolyIn;
 
+// TODO: Workaround for missing console.debug on iOS
+console.debug = console.debug || console.log;
+
 export function simpleJavaScriptCall(): void {
     console.log("simpleJavaScriptCall()");
     return;
 }
 
+export async function awaitPodObject(): void {
+    console.log("awaitPodObject()");
```

Comment:
Apparently, you can obtain the function name inside the function by using `arguments.callee.name`. Instead of repeating the name in two places, maybe use a single source of truth for the name which is going to be the function itself.

Reply:
(This would be a totally amazing use case for aspects)

Reply:
... or simply a function that logs the call and calls...

Reply:
It's not needed anymore with my refactoring - though I left these in redundantly for now... I'm fixing the test feature up to work again, I have no expectation that the old code is gonna be amazing :D But for future reference, `arguments.callee` is deprecated.